tests/bluetooth/host/buf/bt_buf_get_evt/src/main.c

```diff
@@ -92,14 +92,17 @@ static struct net_buf_pool *get_memory_pool(bool discardable)
  */
 ZTEST(bt_buf_get_evt_default_events_returns_not_null, test_returns_not_null)
 {
-	static struct net_buf expected_buf;
+#define USER_DATA_SIZE sizeof(struct bt_buf_data)
```

Comment:
This is a rather strange place to define a macro, especially because its scope is not limited to this function.

Please move the macro definition to the top of the file, or make it `const size_t user_data_size = sizeof(struct bt_buf_data)` instead.